test/Toggle/IsCurrentUserToggleActiveTest.php

```diff
@@ -121,6 +136,65 @@ public function WhenToggleInvisibleAndNotEnabledByPolicyButReleaseDateIsTodayOrI
     }
 
     /**
+     * @test
+     */
+    public function GivenToggleInvisibleAndNotSetByPolicyAndReleaseDateIsInTheFutureAndUserIsAutoSubscribed_ThenInactive()
+    {
+        $this->setupChecker( false, false, false, false, false, false, false );
```

Comment:
Yeah I don't like it either at all :S I just didn't want to refactor all the tests and I saw all of them were using this, so I kept using it too. Should I refactor the whole file?